ui/components/app/flask/snap-content-footer/index.scss

```diff
@@ -0,0 +1,15 @@
+.snap-content-footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 16px 0;
+
+  .typography {
+    font-size: 12px;
+  }
```

Comment:
[REQ] you should use the `variant` prop on `Typography` instead of `as` as is for specifying the html tag, not the styles. the variant you're looking for is H7